### PR TITLE
BNF-1196 and BNF-1318 Site distinction banner and homepage breadcrumbs

### PR DESCRIPTION
**Pull request by ediblecode**
https://nicedigital.atlassian.net/browse/BNF-1196
https://nicedigital.atlassian.net/browse/BNF-1318

### CODE REVIEW

File: gatsby/src/components/SiteDistinction/SiteDistinction.module.scss [hunk 1]
```diff
@@ -0,0 +1,135 @@
+@use '@nice-digital/nds-core/scss/core';
+@import '../../styles/settings';
+
+.wrapper {
+  border-top: core.rem(core.$nds-spacing-small) solid #000;
```
Comment (barkertron) [inline gatsby/src/components/SiteDistinction/SiteDistinction.module.scss:5]: Sorry for pedantry - should #000 be replaced by an NDS colour though?

**Review by barkertron — DISMISSED** [review]

File: gatsby/src/components/SiteDistinction/SiteDistinction.module.scss [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+  text-shadow: 1px 0 0 currentColor;
+  top: -1px; // To avoid a 1px gap in Chrome
+
+  &:hover {
+    color: core.$nds-colour-true-black;
+  }
+}
+
+.currentSiteTab {
+  z-index: 1; // To site above the other site tab
```
Comment (johndavey72) [inline gatsby/src/components/SiteDistinction/SiteDistinction.module.scss:58]: sit above or site above? Either works I guess...?

File: gatsby/src/components/SiteDistinction/SiteDistinction.test.tsx [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+					name: `switch to ${otherSiteTitleShort}`,
+				});
+				expect(link).toHaveAttribute(
+					"data-tracking",
+					`${otherSiteTitleShort.toLowerCase()}-link`
+				);
+			});
+		});
+
+		describe("Visually hidden", () => {
```
Comment (johndavey72) [inline gatsby/src/components/SiteDistinction/SiteDistinction.test.tsx:155]: There's no test for the visually hidden "currently viewing..." - I couldn't get hold of it without node access, is it worth adding one if we can?

File: gatsby/src/components/SiteDistinction/SiteDistinction.module.scss (hunk 2 above)
Comment (ediblecode) [reply gatsby/src/components/SiteDistinction/SiteDistinction.module.scss:58]: Should be sit!

File: gatsby/src/components/OnThisPage/OnThisPage.module.scss [hunk 4]
```diff
@@ -1,11 +1,14 @@
 @use '@nice-digital/nds-core/scss/core';
 
+$site-distinction-banner-clearance: core.rem(core.$nds-spacing-large * 2);
+
 .wrapper {
   @include core.nds-hide-print;
+  top: core.rem(3rem);
```
Comment (barkertron) [inline gatsby/src/components/OnThisPage/OnThisPage.module.scss:7]: Prob don't need to use `core.rem` if the unit is already in rem?

**Review by barkertron — APPROVED** [review]

**Pull request merged**